modules/ui/fields/combo.js

```diff
@@ -190,6 +192,37 @@ export function uiFieldCombo(field, context) {
     }
 
     function getOptions(allOptions) {
+        // Get dropdown list for language: key via localizer instead of taginfo
+        if (field.key === 'language:') {
+          let codes = Object.keys(localizer.languages());
+
+          let options = codes.map((c) => {
+            let name = localizer.languageName(c);
+            return {
+              key: c,
+              value: name,
+              title: name,
+              display: selection => selection.text(name)
+            };
+          }).filter(Boolean);
+
+          options.sort((a, b) => {
+            return a.value.localeCompare(b.value);
+          });
+
+          // inserting others because it does not come via _dataLanguages
+          options.push({
+            key: 'others',
+            value: 'others',
+            title: 'others',
```

Comment:
We should allow id-tagging-schema to provide its own localized string for the title of this option. Can we make the initialization of this option more consistent with the generic initialization down below?